scipy/ndimage/filters.py

```diff
@@ -305,7 +305,12 @@ def gaussian_filter(input, sigma, order=0, output=None,
            [20, 22, 24, 25, 27],
            [29, 31, 33, 34, 36],
            [35, 37, 39, 40, 42]])
-
+    >>> from scipy import misc
```

Comment:
Should probably add a line break here so that the two examples don't run together.